README.md

```diff
@@ -683,6 +683,7 @@ Table of Contents
   * [vatlayer.com](https://vatlayer.com/) — Instant VAT number validation and EU VAT rates API, free 100 API requests/month
   * [fraudlabspro.com](https://www.fraudlabspro.com) — Help merchants to prevent payment fraud and chargebacks. Free Micro Plan available with 500 queries/month.
   * [currencystack.io](https://currencystack.io/) — Production-ready real-time exchange rates for 154 currencies.
+  * [routefusion.co](https://routefusion.co/) — Developer focused cross borders bank-to-bank payments API.  Programatically send money to 180+ countries.  Go live and begin transacting cross border bank to bank payment in your app in under a month with 0 cost and risk.
```

Comment:
I do not see any information at all about costs associated with this service? Someone is paying, how and when?

Reply:
To develop and release code into production is completely free.  We make our money off of the exchange rate spreads, just like any cross border app.  The biggest difference is we are honest with the process, do not charge a monthly fee, or charge any type of integration fee.  We see ourselves as helping developers actually build new and innovative solutions without making them pay for it, like all of the competitors.